common/lib/xmodule/xmodule/js/src/capa/schematic.js

```diff
@@ -2147,7 +2147,7 @@ schematic = (function() {
 	    this.bg_image.height = this.height;
 
 	    if (!this.diagram_only) {
-		this.canvas.tabIndex = 1; // so we get keystrokes
+        this.canvas.tabIndex = 0; // so we get keystrokes
```

Comment:
For reference later--- This is just because the indentation is inconsistent in this file.  The changed line is using the correct convention.